### ISSUE TITLE
Update the ‘How do I find the information I’m looking for?’ section

### ISSUE THREAD

**Issue by JasonKleinert** [TNRIS/iswp2017#239]
10. Update the ‘How do I find the information I’m looking for?’ section with the following text and images:
A good place to start is the ‘View data for’ drop down menu found in the upper left of home page. From this drop down you can navigate to pages with state water plan data organized by a specific area of interest or data type.

Once you’ve navigated to a specific page you can start exploring the data related to that topic/area. Each page will contain demand, existing supply, need (potential shortage), and strategy supply data related to that page topic. The data are presented in a variety of formats such as graphs, figures, tables, and maps. Many of the page elements are interactive so you can hover over and click elements to show more detail, use embedded search and sort functions to locate information, or click hyperlinks to navigate to other pages in the application.
If you are interested in digging deeper and interacting more with the data on a page, scroll down to the ‘Raw Data’ section. In addition to reflecting the decade and theme button selections in the banner above the map, the data displayed in this section can be filtered and sorted using the drop-down boxes below the ‘Raw Data’ section heading. The column toggles allow you to control the aggregation levels of the data.
To change an existing column, update that selected drop-down to another item in the list
To remove an existing column, update that selected drop-down to the blank entry in the list
To add a new column, select it from the ‘Sub Dimension…’ drop down list

Finally, data is available for you to download as Comma-Separated Value (.csv) files through links at the bottom of each page. Downloads of statewide datasets are available at the bottom of the application homepage, and all water volumes are in units of acre-feet/year. For any questions regarding the data or how to apply it, please contact: WRPdatarequests@twdb.texas.gov.

**Comment by JasonKleinert** [comment]
Should this section include and replace the current Download Data section?

**Comment by lebell** [comment]
@JasonKleinert - Thanks for bringing this up! We don't want to replace the Download Data section currently because we like that users can quickly jump to that section if that's all they're interested in, but looking at this again we do want to change the plan for both these sections a bit. If possible could we:

1. Remove the Download Data graphic in this section (looking at this again it seems confusing to show a screenshot of unclickable links).

2. Replace everything in the current Download Data section up until he API piece with "Data is available for you to download as Comma-Separated Value (.csv) files through links at the bottom of each page of the application. Downloads of statewide datasets are available at the bottom of the application homepage, and all water volumes are in units of acre-feet/year. For any questions regarding the data or how to apply it, please contact: WRPdatarequests@twdb.texas.gov."

Also, I did not include any type of specific formatting in the document so please style the content to match or look nice on the page as you see fit. Thank you!

**Comment by JasonKleinert** [comment]
So, to verify, remove the links from the current Downloading data section and replace with the block text above?

**Comment by lebell** [comment]
Yes, but please add spacing as necessary to break up the block text if it looks too busy once you add it to the page.

**Comment by JasonKleinert** [comment]
Ready for review at : https://2017-staging.texasstatewaterplan.org/statewide

**Comment by lebell** [comment]
Could you please make a small update to the Download Data section text of adding the word "application" ahead of "page" at the end of the first sentence? So it will read "Data is available for you to download as Comma-Separated Value (.csv) files through links at the bottom of each application page." Otherwise everything looks great- thank you!

**Comment by lebell** [comment]
Looks good, thank you! Please mark as done.
